### ISSUE TITLE
False positive: unused import for TypeVar bounds

### ISSUE THREAD

**Issue by rraval** [PyCQA/pyflakes#572]
Liberated from #571.

```python
from typing import TYPE_CHECKING, Generic, TypeVar

if TYPE_CHECKING:
    # imagine this being some module that has circular imports
    from threading import Thread  # <--- pyflakes: 'threading.Thread' imported but unused

T = TypeVar("T", bound="Thread")

class Base(Generic[T]):
    pass
```

Version information:

```
$ pyflakes --version
2.2.0 Python 3.6.8 on Linux
```

**Comment by bluetech** [comment]
This will be fixed by #534

**Comment by asottile** [comment]
Via #534
